common/persistence/sql/sqlTaskManager.go

```diff
@@ -431,12 +420,12 @@ func (m *sqlTaskManager) CompleteTasksLessThan(request *persistence.CompleteTask
 	return int(nRows), nil
 }
 
-func (m *sqlTaskManager) shardID(domainID string, name string) int {
-	id := farm.Hash32([]byte(domainID+"_"+name)) % uint32(m.nShards)
+func (m *sqlTaskManager) shardID(domainID primitives.UUID, name string) int {
+	id := farm.Hash32(append(domainID, []byte("_"+name)...)) % uint32(m.nShards)
```

Comment:
This is possibly new behavior in that domainID is now straight bytes rather than the string representation that was previously encoded in UTF-8 when cast to a []byte here.

Reply:
Looks good to me.